package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.1.1",
   "description": "Library to retrieve RSA public keys from a JWKS endpoint",
   "main": "lib/index.js",
+  "types": "index.d.ts",
   "dependencies": {
+    "@types/express-jwt": "0.0.34",
```

Comment:
DevDependencies of dependencies are not installed along. That is, if I have a TypeScript project that has `jwks-rsa` as a `dependency`, `@types/express-jwt` will not get installed if it is defined as a `devDependency` instead of as a `denpendency`. Then the project won't compile and the developer will have to manually install `@types/express-jwt`.

My first attempt was to leave it as a `devDependency`, but it didn't work. Also, the [publishing page](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) on the TypeScript documentation states that the developer has to:

> Make sure all the declaration packages you depend on are marked appropriately in the "dependencies" section in your package.json.